docs/security_checks_guide.adoc

```diff
@@ -0,0 +1,56 @@
+= Verifying Self-Defence Security Check Results utilizing the metrics service
+
+This guide shows how to use Security Checks metrics to gather valueable information regarding the shape of the environment users run your mobile apps in, in relation to security risks that can lead to attack vectors against the data that is stored in-device.
```

Comment:
Also against data transmitted to/from services.
e.g. if the device is rooted, it may be possible to intercept network traffic

@wei-lee would that be right?

Reply:
yes, it is possible, but probably tricky to do. The main concern is the security of the data that is saved on the device.

Reply:
OK. If you're happy with the current wording @wei-lee, then I'm happy